tests/phpunit/elementor/data/v2/base/mock/template/traits/mock-items.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Data\V2\Base\Mock\Template\Traits;
+
+trait Mock_Items {
+	use Mock_Test_Data;
+
+	public function get_items( $request ) {
```

Comment:
Try to wrap with a reflection class.

Reply:
In which method or where I can warp it in Trait?